src/main/java/com/quirko/gui/GuiController.java

```diff
@@ -99,7 +117,23 @@ public void handle(KeyEvent keyEvent) {
                 if (keyEvent.getCode() == KeyCode.P) {
                     pauseButton.selectedProperty().setValue(!pauseButton.selectedProperty().getValue());
                 }
-
+                //different acculator for each difficulty level
+                if(diff == DifficultyType.EASY){
+                    timeLine.setRate(easy_rate);
+                    if(easy_rate < 1.5)
+                        easy_rate+=rate*easy_acc;
+                }
+                else if(diff == DifficultyType.MEDIUM){
+                    timeLine.setRate(medium_rate);
+                    if(medium_rate < 2)
+                        medium_rate+=rate*medium_acc;

```

Comment:
use proper spacing between terms (also same applies for other places in this file)